src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1089,6 +1093,12 @@ public ErrorDiagnostic RuntimeValueNotAllowedInVariableForBody(string variableNa
                DiagnosticLevel.Info,
                "BCP185",
                $"Encoding mismatch. File was loaded with '{detectedEncoding}' encoding.");
+
+            public Diagnostic FallbackPropertyUsed(string property) => new(
+                TextSpan,
+                DiagnosticLevel.Warning,
+                "BCP186",
+                $"The property \"{property}\" does not exist in the resource definition, although it might still be valid. If this is an inaccuracy in the documentation, please report it to the Bicep Team: https://aka.ms/bicep-type-issues");
```

Comment:
Terminate this string with a "."

Reply:
done + moved link from plain text into a clickable option.